### PR TITLE
Adding a Basic Sign Up Page With Working Local Storage

### PR DESCRIPTION
**Pull request by MasoodAkhtar1993**

### CODE REVIEW

File: Sign Up Page/signup.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./signup.css" />
+    <title>Sign Up Page</title>
+  </head>
+  <body>
+    <div class="maincon">
+      <h1>Welcome</h1>
+      <h2>Create Account</h2>
+      <form action="./SignUp.html">
```
Comment (Asadfaraz) [inline Sign Up Page/signup.html:21]: Can you please explain why you write `action="./SignUp.html"` inside the form?

File: Sign Up Page/signup.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        </div>
+
+        <div class="passcon">
+          <input
+            name="password"
+            required
+            id="password"
+            placeholder="Password"
+            type="password"
+            autocomplete="off"
```
Comment (Asadfaraz) [inline Sign Up Page/signup.html:56]: in every input the autocomplete option is by default off so why are you writing it again and again?

**Review by Asadfaraz — CHANGES_REQUESTED** [review]
Please go through my comments on the HTML page.

Also please include comments on every page because it is the best way to work in a team so others can easily understand your content.

File: Sign Up Page/signup.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+  #user_name,
+  #email,
+  #password,
+  #confirm_password {
+    border-radius: 8px;
+    width: 20em;
+    height: 2em;
+  }
```
Comment (Asadfaraz) [inline Sign Up Page/signup.css]: You can also make the same class for these elements and besides targeting here every element individually you can use that class.

File: Sign Up Page/signup.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
```
Comment (Asadfaraz) [inline Sign Up Page/signup.css]: You can add a single class to these elements instead of accessing them via IDEs

File: Sign Up Page/signup.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    this.name = name;
+    this.email = email;
+  }
+}
+
+signUpBtn.addEventListener("click", function () {
+  const user = new User(Date.now(), nameInput.value, emailInput.value);
+  users.push(user);
+  const usersJson = JSON.stringify(users);
+  localStorage.setItem("users", usersJson);
```
Comment (Asadfaraz) [inline Sign Up Page/signup.js:46]: Please show a user an alert box that your registration has successfully submitted or failed!

**Review by Asadfaraz — COMMENTED** [review]
Please read the suggestions

File: Sign Up Page/signup.html (hunk 1 above)
Comment (MasoodAkhtar1993) [reply Sign Up Page/signup.html:21]: Just to write something, here it will go to login page or landing page.

File: Sign Up Page/signup.js (hunk 5 above)
Comment (MasoodAkhtar1993) [reply Sign Up Page/signup.js:46]: Ok will do it

File: Sign Up Page/signup.css (hunk 4 above)
Comment (MasoodAkhtar1993) [reply Sign Up Page/signup.css]: I changed it in recent commits.

File: Sign Up Page/signup.html (hunk 2 above)
Comment (MasoodAkhtar1993) [reply Sign Up Page/signup.html:56]: It was showing the drop down menu of previous inputs which I didn't want as they were just inputs used for testing.

**Pull request merged**